packages/hooks/src/useFetchTokens.ts

```diff
@@ -52,12 +52,13 @@ const fetchTokens = async () => {
   return [] as IToken[];
 };
 
-export const useFetchTokens = () => {
+export const useFetchTokens = ({ enabled = true }: { enabled: boolean }) => {
```

Comment:
As mentioned in the other comment this breaks the existing app. Could you please ensure the smart invoice app builds and works as you update these hooks for DM? :)